Core/State.cs

```diff
@@ -42,17 +43,27 @@ void ProcessUpdates(List<Transaction> ts)
 			}
 		}
 
-		public static TimeSeries GetTimeSeries(string category, double smoothingRatio)
+		public static SmoothedTimeSeries GetSmoothedTimeSeries(string category, double smoothingRatio)
 		{
 			var filteredTransactions = Transactions.Where(categoryFilters[category]).ToList();
-			return new TimeSeries(filteredTransactions, smoothingRatio);
+			return new SmoothedTimeSeries(filteredTransactions, smoothingRatio);
 		}
 
-		public static TimeSeries GetTimeSeriesUnion(IEnumerable<string> categories, double smoothingRatio)
+		public static CumulativeTimeSeries GetCumulativeTimeSeries(string category, double smoothingRatio)
+		{
+			return new CumulativeTimeSeries(1, 4);
+		}
+
+		public static SmoothedTimeSeries GetSmoothedTimeSeriesUnion(IEnumerable<string> categories, double smoothingRatio)
 		{
 			Func<Transaction, bool> filter = t => categories.Any(c => categoryFilters[c](t));
 			var filteredTransactions = Transactions.Where(filter).ToList();
-			return new TimeSeries(filteredTransactions, smoothingRatio);
+			return new SmoothedTimeSeries(filteredTransactions, smoothingRatio);
+		}
+
+		public static CumulativeTimeSeries GetCumulativeTimeSeriesUnion(IEnumerable<string> categories, double smoothingRatio)
+		{
+			return new CumulativeTimeSeries(1, 4);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assign this magic number '4' to a well-named (variable|constant), and use the (variable|constant) instead.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4992784)